integration-tests/tests/run-notebook.js

```diff
@@ -25,6 +25,7 @@ const testRunNotebookFn = _.flow(
   await select(page, 'Language', 'Python 3')
   await click(page, clickable({ text: 'Create Notebook' }))
   await click(page, clickable({ textContains: notebookName }))
+  await delay(2000)
```

Comment:
Do you need this new delay or is increasing the delay below enough? Adding delays in a test is not ideal. When necessary, it should be justified like the comment below justifies the existing delay. Without that, someone coming across this in the future has no idea if the amount of delay is still appropriate or if it's even needed at all.

Reply:
💯

Reply:
I've been having more success with this new delay locally, but going to try a few runs without it on this PR.